doc/playbook.rst

```diff
@@ -79,66 +87,71 @@ From the ``main`` branch you can release a new major version (``X.0.0``), a new
-In practice, almost any Science Pipelines update is at least a minor version, because new features are added constantly.
-In the future, there may be "patched weekly" builds, which would justify a patch version of Prompt Processing.
+   If you are planning to update the Science Pipelines tag, you should also check the `Science Pipelines changelog <https://lsst-dm.github.io/lsst_git_changelog/weekly/>`_.
+   In practice, almost any Science Pipelines update is at least a minor version, because new features are added constantly.
+   You may need to do a "Quick-Stack" build, or build the science pipelines manually. This would justify a patch version of Prompt Processing.
 
-For the ``prompt_processing`` service, a new major version is triggered by any of the following:
+   **The following changes will always trigger a major release:**
 
-* Incompatibility with old fanned-out ``nextVisit`` messages (almost any change to ``Visit`` qualifies)
-* Incompatibility with an old `APDB schema`_, `ApdbSql`_, `ApdbCassandra`_, or `ApdbCassandraReplica`_ version (see `DMTN-269`_ for the distinction)
-* Incompatibility with an old `Butler dimensions-config`_ version
-* A new major version of the `Alerts schema`_ (see `DMTN-093`_ for details)
+   For the ``prompt_processing`` service:
 
-For the `next_visit_fan_out`_ service, a new major version is triggered by any of the following:
+   * Incompatibility with old fanned-out ``nextVisit`` messages (almost any change to ``Visit`` qualifies)
+   * Incompatibility with an old `APDB schema`_, `ApdbSql`_, or `ApdbCassandra`_ version (see `DMTN-269`_ for the distinction)
+   * Incompatibility with an old `Butler dimensions-config`_ version
+   * A new major version of the `Alerts schema`_ (see `DMTN-093`_ for details)
 
-* Incompatibility with old Summit ``nextVisit`` messages
-* Breaking changes in the fanned-out ``nextVisit`` messages (almost any change to ``NextVisitModel`` qualifies)
+   For the `next_visit_fan_out`_ service:
 
-2. Create a Release
+   * Incompatibility with old Summit ``nextVisit`` messages
+   * Breaking changes in the fanned-out ``nextVisit`` messages (almost any change to ``NextVisitModel`` qualifies)
 
-On GitHub.com, navigate to the main page of the repository.
-To the right of the list of files, click **Releases**.
-At the top of the page, click **Draft a new release**.
-Type a tag using semantic versioning described in the previous section.
-The Target should be the main branch.
+#. Create a Release
 
-Select **Generate Release Notes**.
-This will generate a list of commit summaries and of submitters.
-Add text as follows.
+   On GitHub.com, navigate to the main page of the repository.
+   To the right of the list of files, click **Releases**.
+   At the top of the page, click **Draft a new release**.
+   Type a tag using semantic versioning described in the previous section.
+   The Target should be the main branch.
```

Comment:
I finally figured out what sometimes causes "Previous tag: auto" to pick the wrong version -- when there aren't any (eligible) PRs since the last tag, it goes back to the version before the most recent PR.

Add a note warning that `auto` is unreliable if there are no changes in PP itself?